driver/command/command_nix_test.go

```diff
@@ -174,3 +178,40 @@ func TestCommandDriverOutputs(t *testing.T) {
 	}
 	CreateAndRunTestCommandDriver(t, name, content, testfunc)
 }
+
+func TestCommandDriverCancellation(t *testing.T) {
+	content := `#!/bin/sh
+		echo command executed
+	`
+	name := "test-command.sh"
+	output := bytes.Buffer{}
+	testfunc := func(t *testing.T, cmddriver *Driver) {
+		if !cmddriver.CheckDriverExists() {
+			t.Fatalf("Expected driver %s to exist Driver Name %s ", name, cmddriver.Name)
+		}
+		op := driver.Operation{
+			Action:       "install",
+			Installation: "test",
+			Parameters:   map[string]interface{}{},
+			Image: bundle.InvocationImage{
+				BaseImage: bundle.BaseImage{
+					Image:     "cnab/helloworld:latest",
+					ImageType: "docker",
+				},
+			},
+			Revision:    "01DDY0MT808KX0GGZ6SMXN4TW",
+			Environment: map[string]string{},
+			Files: map[string]string{
+				"/cnab/app/image-map.json": "{}",
+			},
+			Out:    &output,
+			Bundle: &bundle.Bundle{Name: "mybun"},
+		}
+		ctx, cancel := context.WithCancel(context.Background())
+		cancel()
```

Comment:
We probably want this instead?
```suggestion
		defer cancel()
```